python/ray/node.py

```diff
@@ -48,7 +52,11 @@ def __init__(self, ray_params, head=False, shutdown_at_exit=True):
             shutdown_at_exit (bool): If true, a handler will be registered to
                 shutdown the processes started here when the Python interpreter
                 exits.
+            connect_only (bool): If true, connect to the node without starting
+                new processes.
         """
+        assert not (shutdown_at_exit and connect_only), (
```

Comment:
This should raise a `ValueError` and should not be an `assert`. Assert should only be used for things that we believe will never happen. `ValueError` should be used if someone simply enters the arguments incorrectly. Could you change this in a subsequent PR?